security-privacy-questionnaire.md

```diff
@@ -38,8 +38,12 @@ after a browser prompt and user consent.
 
 ## 5. Do the features in your specification introduce new state for an origin that persists across browsing sessions? 
 
-Secure Payment Confirmation relies on Web Authentication, which introduces state. Secure Payment Confirmation
-does not introduce additional state.
+Secure Payment Confirmation relies on Web Authentication, which introduces state. Secure Payment Confirmation does not introduce additional state.
+
+Secure Payment Confirmation includes an "opt-out" feature: an
```

Comment:
We do not persist any state via opt-out, so I'm not sure of its relevance in this question?

Reply:
Indeed.